tests/unit/python/foglamp/services/core/api/test_browser_assets.py

```diff
@@ -317,6 +317,7 @@ def q_result(*args):
             # FIXME: ordering issue and add tests for datetimeunits request param
             # assert '{"aggregate": [{"operation": "min", "json": {"column": "reading", "properties": "humidity"}, "alias": "min"}, {"operation": "max", "json": {"column": "reading", "properties": "humidity"}, "alias": "max"}, {"operation": "avg", "json": {"column": "reading", "properties": "humidity"}, "alias": "average"}], "where": {"column": "asset_code", "condition": "=", "value": "fogbench_humidity"}, "limit": 20}' in args1
 
+    @pytest.mark.skip(reason='Test skip')
```

Comment:
As these tests needs a rewrite once api is mature enough for the requirements. We should create a JIRA for this and `@pytest.mark.skip(reason='TODO: FOGL-xxx summary of JIRA')` for all these bucket api tests.

Reply:
FOGL-3541 created